python-package/xgboost/spark/estimator.py

```diff
@@ -355,39 +444,55 @@ class SparkXGBRanker(_SparkXGBEstimator):
     Parameters
     ----------
 
-    callbacks:
-        The export and import of the callback functions are at best effort. For
-        details, see :py:attr:`xgboost.spark.SparkXGBRanker.callbacks` param doc.
-    validation_indicator_col:
-        For params related to `xgboost.XGBRanker` training with
+    features_col:
+        When the value is string, it requires the features column name to be vector type.
+        When the value is a list of string, it requires all the feature columns to be numeric types.
+    label_col:
+        Label column name. Default to "label".
+    prediction_col:
+        Prediction column name. Default to "prediction"
+    pred_contrib_col:
+        Contribution prediction column name.
+    validation_indicator_col
+        For params related to `xgboost.XGBClassifier` training with
         evaluation dataset's supervision,
-        set :py:attr:`xgboost.spark.XGBRanker.validation_indicator_col`
-        parameter instead of setting the `eval_set` parameter in `xgboost.XGBRanker`
+        set :py:attr:`xgboost.spark.SparkXGBClassifier.validation_indicator_col`
+        parameter instead of setting the `eval_set` parameter in `xgboost.XGBClassifier`
         fit method.
     weight_col:
         To specify the weight of the training and validation dataset, set
-        :py:attr:`xgboost.spark.SparkXGBRanker.weight_col` parameter instead of setting
-        `sample_weight` and `sample_weight_eval_set` parameter in `xgboost.XGBRanker`
-        fit method.
-    xgb_model:
-        Set the value to be the instance returned by
-        :func:`xgboost.spark.SparkXGBRankerModel.get_booster`.
+        :py:attr:`xgboost.spark.SparkXGBClassifier.weight_col` parameter instead of setting
+        `sample_weight` and `sample_weight_eval_set` parameter in `xgboost.XGBClassifier`
+        fit method.    base_margin_col
+        Base margin column name
+    base_margin_col:
+        To specify the base margins of the training and validation
+        dataset, set :py:attr:`xgboost.spark.SparkXGBClassifier.base_margin_col` parameter
+        instead of setting `base_margin` and `base_margin_eval_set` in the
+        `xgboost.XGBClassifier` fit method. Note: this isn't available for distributed
```

Comment:
```suggestion
        :py:class:`xgboost.XGBClassifier` fit method. Note: this isn't available for distributed
```

Reply:
What do you mean by isn't available for distributed training?

Reply:
Done. removed it.